src/styles/_global.sass

```diff
@@ -42,6 +42,22 @@ textarea
 	font-size: inherit
 	line-height: inherit
 
+button
+	border: none
+	background: transparent
+	padding: 0
+	margin: 0
+	font-weight: inherit
+	text-align: inherit
+	color: inherit
+	cursor: pointer
+	text-transform: inherit
+
+a,
+button
+	&:focus
+		outline: none
```

Comment:
Stále mi tento přístup - odstraníme to všude a kodér to dostyluje všude správně - přijde nevhodný. Pro aktivní navigaci po stránce pouze klávesnicí potřebuji vidět, na jakém aktivním prvku se právě nacházím. S touto změnou se má pozice stává neviditelnou a web, pokud kodér správně nenastyluje na všechny aktivní prvky `:focus` stav, ovládáním bez myši nepřístupný.
Dle osobních zkušeností se vždy na nějaký prvek zapomene (např. odkaz v textu článku) a nechtěně tak vyřadí z navigace po stránce.

Navrhuji tedy outline nerušit globálně, jen pouze u `.button` komponenty, která toto ošetřeno má.